lib/app.js

```diff
@@ -17,6 +17,8 @@ const devMode = argv.dev || options.http
 const version = require('../package.json').version
 logger.info('Version: %s', version)
 
+let settings
```

Comment:
I don't think this has to be a global mutable variable. See more comments, below.